lib/matplotlib/axes/_base.py

```diff
@@ -3074,25 +3074,25 @@ def set_xlim(self, left=None, right=None, emit=True, auto=False,
             The left xlim (default: None, which leaves the left limit
             unchanged).
             The left and right xlims may be passed as the tuple
-            (`left`, `right`) as the first positional argument (or as
-            the `left` keyword argument).
+            (*left*, *right*) as the first positional argument (or as
+            the *left* keyword argument).
 
         right : scalar, optional
-            The right xlim (default: None, which leaves the right limit
+            The right xlim (default: ``None``, which leaves the right limit
```

Comment:
We don't have a consistent policy for None, True, False so far.

There are all styles present in our docs:
- plain
- literal: double backticks
- italics: starred

Personally, I find double backslashes too cumbersome to type and visually too broad in plain text for there short words. I just use `*None*`.

The [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) and [sphinx](http://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) examples just use plain text.

Just mentioning. Since we currently have no standards, I won't hold this up.

Reply:
Happy to adopt whatever standard folks like the best, or to start a standard!  It was plain before, happy to put it back.